apps/activityreminder/metadata.json

```diff
@@ -17,6 +17,6 @@
     {"name": "activityreminder.img", "url": "app-icon.js", "evaluate": true}
   ],
   "data": [
-    {"name": "ar.settings.json", "name":  "ar.stepsarray.json"}
+    {"name": "activityreminder.s.json", "name":  "activityreminder.sa.json"}
```

Comment:
Hi - I'm afraid this won't work properly - I think you need `{"name": "activityreminder.s.json"},{"name": "activityreminder.sa.json"}` ?

Reply:
Thanks, I corrected it ;)